### DISCUSSION TITLE
kangaroo/best-practice/managed-harbor-select-access-type

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2184] [Announcements]
# kangaroo/best-practice/managed-harbor-select-access-type

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kangaroo/best-practice/managed-harbor-select-access-type.html

**Comment by liyuerich** [comment]
前提条件：

集群必须安装 Ingress 组件，Ingress 组件可以是 Nginx、Traefik、HAProxy 等，具体安装方法请参考 Ingress。
集群必须安装 DNS 服务，DNS 服务可以是 CoreDNS、KubeDNS、Bind 等，具体安装方法请参考 DNS。
集群必须安装 LoadBalancer 组件，LoadBalancer 组件可以是 MetalLB、F5、Nginx 等，具体安装方法请参考 LoadBalancer。


ingress 和LB 都要装吗？ 如果都是Nginx，是不是只有一个就行？
